src/client/grid/hex.tsx

```diff
@@ -85,6 +85,8 @@ function landColorStyle(space: Land): string {
       return styles.darkMountain;
     case SpaceType.RIVER_MOUNTAIN:
       return styles.riverMountain;
+    case SpaceType.POLAND_DARK_MOUNTAIN:
```

Comment:
Why have POLAND_DARK_MOUNTAIN? Can't you just use DARK_MOUNTAIN and change the cost?